tedana/stats.py

```diff
@@ -152,3 +153,76 @@ def get_coeffs(data, X, mask=None, add_const=False):
         betas = utils.unmask(betas, mask)
 
     return betas
+
+
+@due.dcite(BibTeX("""
+           @article{hughett2007accurate,
+             title={Accurate Computation of the F-to-z and t-to-z Transforms
+                    for Large Arguments},
+             author={Hughett, Paul and others},
+             journal={Journal of Statistical Software},
+             volume={23},
+             number={1},
+             pages={1--5},
+             year={2007},
+             publisher={Foundation for Open Access Statistics}
+           }
+           """),
+           description='Introduces T-to-Z transform.')
+@due.dcite(Doi('10.5281/zenodo.32508'),
+           description='Python implementation of T-to-Z transform.')
+def t_to_z(t_values, dof):
+    """
+    Convert t-values to z-values.
+
+    Parameters
+    ----------
+    t_values
+    dof
+
+    Returns
+    -------
+    out
+
+    Notes
+    -----
+    From Vanessa Sochat's TtoZ package.
```

Comment:
Add the package link in this note and possibly also in the reference or report logger: https://github.com/vsoch/TtoZ